### PR TITLE
[RW-6488][risk=low]: Start populating googleProject from Firecloud for PPW 

### PR DESCRIPTION
**Pull request by yonghaoy**
Description:
Tested locally, verified that we can create then get workspace successfully with the correct field in DB. 

Backfilled
mysql> select google_project, workspace_namespace from workspace;
+------------------------+------------------------+
| google_project         | workspace_namespace    |
+------------------------+------------------------+
| aou-rw-local1-38323f45 | aou-rw-local1-38323f45 |
| aou-rw-local1-3f8cdeb2 | aou-rw-local1-3f8cdeb2 |
| aou-rw-local1-d8fbcad1 | aou-rw-local1-d8fbcad1 |
| aou-rw-local1-719fcb68 | aou-rw-local1-719fcb68 |
| aou-rw-local1-6fceafc3 | aou-rw-local1-6fceafc3 |
| aou-rw-local1-acc779ad | aou-rw-local1-acc779ad |
| aou-rw-local1-bdb1879d | aou-rw-local1-bdb1879d |
| aou-rw-local1-d0aed2f9 | aou-rw-local1-d0aed2f9 |
| aou-rw-local1-00436439 | aou-rw-local1-00436439 |
| aou-rw-local1-4ab75dbd | aou-rw-local1-4ab75dbd |
| aou-rw-local1-dda8418b | aou-rw-local1-dda8418b |
| aou-rw-local1-4b1aa921 | aou-rw-local1-4b1aa921 |
+------------------------+------------------------+

Next steps:
1. Adding integration test
2. Backfill google_project field
3. Make this DB column non-null.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Comment by jmthibault79** [comment]
looks good so far

File: api/db/changelog/db.changelog-master.xml [hunk 1]
```diff
@@ -166,6 +166,7 @@
   <include file="changelog/db.changelog-154-wgs-cromwell-extract-submission.xml"/>
   <include file="changelog/db.changelog-155-add-ras-link-entries.xml"/>
   <include file="changelog/db.changelog-156-billing-buffer-entry-status-index.xml"/>
+  <include file="changelog/db.changelog-157-workspace-add-google-project.xml"/>
```
Comment (jmthibault79) [inline api/db/changelog/db.changelog-master.xml:169]: FYI: I just merged a new entry here so the number will have to increase.

Unfortunately we don't have a good process for these kinds of conflicts.

Comment (yonghaoy) [reply api/db/changelog/db.changelog-master.xml]: merged

**Comment by yonghaoy** [comment]
unit-test keep OOM

File: api/db/changelog/db.changelog-158-workspace-add-google-project.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      <column name="google_project" type="varchar(255)">
+        <constraints nullable="true"/>
+      </column>
+    </addColumn>
+
+    <!--
+    Backfill google_project to the workspace.namespace
+    -->
+    <sql>
+      UPDATE workspace
```
Comment (jmthibault79) [inline api/db/changelog/db.changelog-158-workspace-add-google-project.xml:18]: nice and simple!

**Review by jmthibault79 — APPROVED** [review]

**Review by calbach — APPROVED** [review]

**Pull request merged**